src/test/groovy/life/qbic/datamodel/dtos/business/CustomerSpec.groovy

```diff
@@ -0,0 +1,21 @@
+package life.qbic.datamodel.dtos.business
+
+import spock.lang.Specification
+
+/**
+ * Simple tests for the customer dto class
+ *
+ * @author Sven Fillinger
+ * @since 1.11.0
+ */
+class CustomerSpec extends Specification{
+
+  def "wrong email shall raisa a validation exception"() {
+    when:
+    def customer = new Customer(null,"","","test",[])
+
```

Comment:
I think we should also test the other properties of the customer DTO since it's a key component of qOffer 2.0.